.storybook/viewport.ts

```diff
@@ -49,6 +49,19 @@ const viewportEntries = Object.entries(breakpoints).map(([name, width]) => {
 	];
 });
 
+const extraViewPorts = {
+	largeTablet: {
```

Comment:
Just to make sure I understand, we're adding our own new intermediate breakpoint (`largeTablet`) which doesn't exist in source, between tablet and desktop? Why's that needed in storybook? Looking at the changes below, I think we change the styling to do the column layout from tablet up (rather than desktop up as before)?

Reply:
Hey, yeah so I added in this extra view specifically for story book. IT is supposed to represent the upperbounds (width) of the tablet breakpoint so that you can view a story at the smallest and largest tablet breakpoint bounds.

In this case it is useful given the image asset on the right hand side is displayed in its full width only at the upper bounds of the tablet breakpoint. Basically it makes it easier to review the component .

Reply:
![Screenshot 2024-09-04 at 09 46 13](https://github.com/user-attachments/assets/8d6b2b74-35f9-4202-8b07-4e36927c03a9)

This probably explains it a lot better ... it allows me to add this extra option when selecting the window size for a story